source/partials/_site_footer.html.erb

```diff
@@ -119,9 +119,9 @@
 
       <%#
         These regional links are a test to see if these links will improve our index rate of non-global pages.
-        We only grab the first 5 because we may eventually add 20+ and this would be ugly.
+        We only grab the first 6 because we may eventually add 20+ and this would be ugly.
```

Comment:
😄  Worth double-checking this looks good…

Reply:
I did. Looks OKish...